static/swagger-specs/profile.yaml

```diff
@@ -832,84 +837,248 @@ paths:
         example: UPS1
       - name: relatedEntityId
         in: query
-        description: The ID of the entity that the ExperienceEvents are associated with. Used when looking up ExperienceEvents. For Native XID lookup, use `relatedEntityId=<XID>` and leave `relatedEntityIdNS` absent; For ID:NS lookup, use both `relatedEntityId` and `relatedEntityIdNS` fields.
+        description: The ID of the entity that the ExperienceEvents are associated with. If `schema.name` is `_xdm.context.experienceevent`, this value **must** specify the related profile entity's identity namespace. This value follows the same rules as `entityId`. 
         schema:
           type: string
         example: GtghAUFkdGVzdDE
       - name: relatedEntityIdNS
         in: query
-        description: The Identity Namespace code of the related entity ID of ExperienceEvent. Used when looking up ExperienceEvents. If this field is used, `entityId` cannot be empty.
+        description: If `schema.name` is "_xdm.context.experienceevent", this value must specify the identity namespace for the entity specified in `relatedEntityId`.
         schema:
           type: string
         example: UPS1
       - name: fields
         in: query
-        description: The fields for the model object. By default, all fields will be fetched. Separated by comma.
+        description: Filters the data returned in the response. Use this to specify which schema field values to include in data retrieved. For multiple fields, separate values by a comma with no spaces between.
         schema:
           type: string
         example: person.name.firstName,person.name.lastName
       - name: mergePolicyId
         in: query
-        description: The ID of the merge policy. A merge policy includes information for Identity stitching and key-value XDM object merging. If not present, the default merge policy will be used.
+        description: Identifies the merge policy by which to govern the data returned. If one is not specified in the call, your organization's default for that schema will be used. If no default merge policy has been configured, the default is no profile merge and no identity stitching.
         schema:
           type: string
         example: example-mergePolicy
       - name: startTime
         in: query
-        description: 'The start time of time range filter for ExperienceEvents. Should be at millisecond granularity. Included. Default: From beginning.'
+        description: 'Inclusively specifies the start time to filter the entities (in milliseconds). By default, this is from beginning.'
         schema:
           type: number
         example: "1539838505"
       - name: endTime
         in: query
-        description: 'The end time of time range filter for ExperienceEvents. Should be at millisecond granularity. Excluded. Default: To the end.'
+        description: 'Exclusively specifies the end time to filter entities (in milliseconds). By default, this is to the end.'
```

Comment:
startTime says "Inclusively" and end time says "Exclusively" and i feel like this might be a mistake.

Reply:
Nope - this one is correct. Just a weird quirk where one _is_ inclusive but the other one isn't. I know. Weird stuff.